service/api_plugin_manifest.go

```diff
@@ -73,3 +74,25 @@ func (as *APIServer) manifestLoadHandler(w http.ResponseWriter, r *http.Request)
 
 	gimlet.WriteJSON(w, manifest)
 }
+
+func sameModules(m manifest.Manifest, modules []model.Module) bool {
+	manifestModules := map[string]manifest.Module{}
+	for name, module := range m.Modules {
+		manifestModules[name] = manifest.Module{
+			Branch: module.Branch,
+			Repo:   module.Repo,
+			Owner:  module.Owner,
+		}
+	}
+	projectModules := map[string]manifest.Module{}
+	for _, module := range modules {
+		owner, repo := module.GetRepoOwnerAndName()
+		projectModules[module.Name] = manifest.Module{
+			Branch: module.Branch,
+			Repo:   repo,
+			Owner:  owner,
+		}
+	}
+
+	return reflect.DeepEqual(manifestModules, projectModules)
```

Comment:
I'd implement this as a method on a type alias for []model.Module, and probably do it without reflection (make them sortable, call sort, and iterate through one, and return false if there's a difference.

Reply:
Made an alias on []Module but I like the reflection implementation better. I think it's quite reasonable and I don't see any reason to duplicate this